src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/OffenderBooking.kt

```diff
@@ -149,6 +149,12 @@ data class OffenderBooking(
 
   @OneToMany(mappedBy = "id.offenderBooking", cascade = [CascadeType.ALL], fetch = FetchType.LAZY)
   val profileDetails: MutableList<OffenderProfileDetail> = mutableListOf(),
+
+  @OneToMany(mappedBy = "offenderBooking", cascade = [CascadeType.ALL], fetch = FetchType.LAZY)
+  val images: MutableList<OffenderBookingImage> = mutableListOf(),
+
+  @OneToMany(mappedBy = "id.offenderBooking", cascade = [CascadeType.ALL], fetch = FetchType.LAZY)
+  val identifyingMarks: MutableList<OffenderIdentifyingMark> = mutableListOf(),
```

Comment:
The hierarchy is:

OffenderBooking -> OffenderBookingImage (joined on OFFENDER_BOOK_ID)

OffenderBooking -> OffenderIdentifyingMark -> OffenderIdentifyingMarkImage (joined on OFFENDER_BOOK_ID and ID_MARK_SEQ=IMAGE_OBJECT_ID).

Both images are on table OFFENDER_IMAGES so they subclass OffenderImage and handle the joins in the subclasses.